tools/pip/setup.py

```diff
@@ -27,7 +27,7 @@
 
 if platform.system() == 'Linux':
     sys.argv.append('--universal')
-    sys.argv.append('--plat-name=manylinux1_x86_64')
+    sys.argv.append('--plat-name=manylinux2010_x86_64')
```

Comment:
Looking at the symbols in `libmxnet.so`, for `GLIBCXX` the highest version used is `GLIBCXX_3.4.19 _ZNSt6chrono3_V212system_clock3nowEv` and for `GLIBC` `GLIBC_2.16  __poll_chk`.

Those versions are not allowed in `manylinux2010`. However, they are allowed in `manylinux2014`.

Reference: https://www.python.org/dev/peps/pep-0571/ https://www.python.org/dev/peps/pep-0599/

Until the CD is set up to build correctly for manylinux2010, I suggest to change this line to 2014.